diff --git a/node_modules/@rneui/base/dist/ListItem/components/PadView.js b/node_modules/@rneui/base/dist/ListItem/components/PadView.js
index 0e90243..fc73cd4 100644
--- a/node_modules/@rneui/base/dist/ListItem/components/PadView.js
+++ b/node_modules/@rneui/base/dist/ListItem/components/PadView.js
@@ -17,7 +17,7 @@ export const PadView = (_a) => {
     const length = React.Children.count(children);
     const Container = Component || View;
     return (React.createElement(Container, Object.assign({}, props, { ref: _root, testID: "RNE__LISTITEM__padView" }), React.Children.map(children, (child, index) => child && [
-        child,
-        index !== length - 1 && React.createElement(View, { style: { paddingLeft: pad } }),
+        React.cloneElement(child, { key: `child-${index}` }),
+        index !== length - 1 && React.createElement(View, { key: `separator-${index}`, style: { paddingLeft: pad } }),
     ])));
 };
